packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -1473,6 +1473,14 @@ class _AndroidViewSurfaceState extends State<AndroidViewSurface> {
   @override
   Widget build(BuildContext context) {
     if (widget.controller.requiresViewComposition) {
+      if (widget.controller.useNewHybridComposition) {
+        // TODO(jonahwilliams): make it actually work.
+        return _PlatformLayerBasedAndroidViewSurface(
```

Comment:
Not sure yet if we need a new surface class

Reply:
We will need to configure touch gestures differently. Will do in follow up.

Reply:
Ack